python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/antlr_asset_selection.py

```diff
@@ -108,19 +108,19 @@ def visitKeySubstringExpr(self, ctx: AssetSelectionParser.KeySubstringExprContex
     def visitTagAttributeExpr(self, ctx: AssetSelectionParser.TagAttributeExprContext):
         key = self.visit(ctx.value(0))
         value = self.visit(ctx.value(1)) if ctx.EQUAL() else ""
-        return AssetSelection.tag(key, value)
+        return AssetSelection.tag(key, value, include_sources=self.include_sources)
```

Comment:
noticed this bug while testing

Reply:
Hmmm I can't mention @briantu but just a heads up to @salazarm . Unit test follow up should happen